vignettes/web_only/BCRpipeline.Rmd

```diff
@@ -128,6 +127,8 @@ Available species: "HomoSapiens" (default), "MusMusculus", "BosTaurus", "Camelus
 
 `.min_nuc_outside_cdr3` — filters out reads with low numbers of nucleotides from V or J gene outside of CDR3 region
```

Comment:
filters out the readings with low nucleotide numbers

Reply:
I noticed that we copied this description from older version of germline.R, and in new version there is more clear description of this parameter: `this parameter sets how many nucleotides should have V or J chain outside of CDR3 to be considered good for further alignment`

Reply:
We can also add that "reads with too short chains are filtered out"